README.rst

```diff
@@ -39,7 +39,7 @@ The easiest way to install aws-cli is to use `pip`_ in a ``virtualenv``::
 
 or, if you are not installing in a ``virtualenv``, to install globally::
 
-    $ sudo pip install awscli
+    $ pip install awscli
```

Comment:
This probably won’t work, plus it’s not right in this part of the doc (it’s the same as the command just before).

`pip install --user awscli` could be an alternative that works without virtual env and without messing us the system (or being denied)

Reply:
So should we just remove it or leave it as it is?

Reply:
I don’t understand the question.

`sudo pip install` is bad, `pip install` won’t work.  More thought is needed (e.g. with the `install --user` idea)